invoker/conformance/pom.xml

```diff
@@ -28,6 +28,7 @@
     <dependency>
       <groupId>com.google.cloud.functions</groupId>
       <artifactId>functions-framework-api</artifactId>
+      <version>1.0.5-SNAPSHOT</version>
```

Comment:
Maybe this and the `function-maven-plugin` version should come from the parent POM?